src/silx/math/fit/test/test_fit.py

```diff
@@ -140,7 +140,7 @@ def testUnconstrainedFitWeight(self):
     def testDerivativeFunction(self):
         parameters_actual = [10.5, 2, 10000.0, 20.0, 150, 5000, 900.0, 300]
         x = numpy.arange(10000.0)
-        y = self.gauss(x, *parameters_actual)
+        y = self.gauss(x, *parameters_actual)  # noqa: F841
```

Comment:
Since its usage is comment, I would remove both this line and the commented usage.
Or eventually comment this line:
```suggestion
        # y = self.gauss(x, *parameters_actual)
```